hibernate-core/src/test/java/org/hibernate/orm/test/filter/DynamicFilterTest.java

```diff
@@ -683,6 +685,7 @@ public void testFilterApplicationOnHqlQueryWithImplicitSubqueryContainingNamedPa
 	}
 
 	@Test
+	@SkipForDialect( dialectClass = InformixDialect.class, majorVersion = 12, matchSubTypes = true, reason = "Informix does not support table alliases in update/delete statements, Issue HHH-18473")
```

Comment:
It seems like `@RequiresDialectFeature(feature = DialectFeatureChecks.SupportsDmlTargetColumnQualifier.class)` with the following implementation might be more appropriate:

```java
	public static class SupportsDmlTargetColumnQualifier implements DialectFeatureCheck {
		public boolean apply(Dialect dialect) {
			return dialect.getDmlTargetColumnQualifierSupport() != DmlTargetColumnQualifierSupport.NONE;
		}
	}
```